Adding parsing of additional model information from sources (#1572)
ie3-institute/PowerSystemDataModel: docs/readthedocs/models/models.md

`operationTime` where `operationStartTime` and `operationEndTime` are both `null` or `""`, the
factory will build an always-on line model.

**Additional Information** <br>

> The input models, excluding time series and values, support additional information. These parameters are read from the data

Reviewer: ```suggestion
The input models, with the exception of time series and values, support additional information. These parameters are read from the data
```

> source and stored as a map inside the models. To retrieve them one can use the method `getAdditionalInformation()`. One

Reviewer: ```suggestion
source and stored as a map within the models. To retrieve them, one can use the method `getAdditionalInformation()`. An
```

> important thing is that these key value pairs are stored as strings. It is the job of the user to handle/convert the data.

Reviewer: ```suggestion
important point is that these key-value pairs are stored as strings. It is the job the user's responsibility to process or convert the data.
```
